conference/react-js/src/reducers/talks.js

```diff
@@ -9,17 +9,19 @@ const talks = (state = [], action) => {
     return payload;
   } else if (type === APPLY_FILTERS) {
     const {
-      filter: { topic, status },
+      filter: { conferenceName, title, status },
```

Comment:
We need some else functionality for filters. This reducer should take different numbers of filter fields